source/IntegrationConfigFactory.ts

```diff
@@ -15,20 +14,20 @@ import { IntegrationType } from 'aws-sdk/clients/apigateway';
  */
 @injectable()
 export default class IntegrationConfigFactory {
-    private _logger: Winston.Logger;
-
     /**
      * Creates specifically typed messages given a json string
      * @param {Container} container The IOC container used to resolve other dependencies
      * @memberof MessageHandlerFactory
      */
-    constructor(@inject(TYPES.Logger) logger: Winston.Logger) {
-        this._logger = logger;
+    constructor(
+        @inject(TYPES.DDLHelper)
+        @named(IntegrationSystemType.Oracle)
```

Comment:
Not sure how I feel about this as an injected constructor parameter since we or someone else may want to toss in their own system type but for now it works. Alternatively resolving directly from the container as part of the create(...) call would keep the constructor clean.